components/Marketplace/MarketplacePage.tsx

```diff
@@ -19,6 +19,13 @@ export default function MarketplacePage() {
     setSearchQuery(event.target.value);
   };
 
+  const handleSortChange = (value: string | null) => {
+    setSortQuery(value);
+    if (value === 'Saved') {
+      toggleRefresh();
+    }
+  };
```

Comment:
it seems we're reusing this set of code every time on save, granted it is extremely small, it could be good to move it into a helper. Although def not needed at our scale! Just suggestion

Reply:
Good idea, prob good to add in after our presentation before submission